SwiftTweaks/TweakTableCell.swift

```diff
@@ -68,15 +68,50 @@ internal final class TweakTableCell: UITableViewCell {
 		imageView.tintColor = AppTheme.Colors.controlSecondary
 		return imageView
 	}()
+    
+    private let optionsTextField: UITextField = {
+        let optionsTextField = UITextField()
+        optionsTextField.textAlignment = .right
+        return optionsTextField
+    }()
+    
+    private let doneButton = UIBarButtonItem(title: "Done TT", style: .done, target: nil, action: nil)
```

Comment:
You cannot change the button target. You should bind to these tweaks. So you get notified when there is a change in a tweaks value.